### DISCUSSION TITLE
Mail Protocol and QUIC

### DISCUSSION THREAD

**Discussion by ghost** [vpnhood/VpnHood#632] [Ideas]
SMTP POP3 IMAP JMAP The government does not block the mail protocol
QUIC `quic://dns.adguard-dns.com`

**Comment by trudyhood** [comment]
Hi,
Would you explain more?

**Reply by ghost** [reply]
bro Is it possible to fake the mail and QUIC protocol?

**Comment by trudyhood** [comment]
1) We are working on QUIC and will add it so VpnHood can establish QUIC tunnels. However, QUIC uses UDP, and many firewalls do let UDP work. 
2) There is no plan to work with POP3 IMAP JMAP on our table at the moment.
